spec/lib/dsc_utils_litmus.rb

```diff
@@ -0,0 +1,125 @@
+def locate_dsc_module(expect_ps_failure)
+  # Init
+  module_paths = run_shell('puppet config print modulepath').stdout.gsub("\n", '').gsub("\r", '').split(';')
+  # Search the available module paths.
+  module_paths.each do |module_path|
+    dsc_module_path = "#{module_path}/dsc".gsub('\\', '/')
+    ps_command = "Test-Path -Type Container -Path '#{dsc_module_path}'"
+    run_shell("powershell.exe -NoProfile -Nologo -Command \"if ( #{ps_command} ) { exit 0 } else { exit 1 }\"", :expect_failures => expect_ps_failure) do |result|
```

Comment:
Feels like this idiom of `"powershell.exe -NoProfile -Nologo -Command \"<whatever Powershell you want to run>\""` (which we have to insert so we can properly call out to Powershell from the cmd prompt) should be moved to Litmus' `run_shell` so we don't have to continually repeat ourselves.